website/content/en/docs/ansible/development-tips.md

```diff
@@ -116,26 +124,25 @@ PLAY [localhost] ***************************************************************
 TASK [Gathering Facts] *********************************************************************
 ok: [localhost]
 
-Task [Foo : set example-memcached namespace to present]
+Task [foo : set example-coonfig configmap to present]
 changed: [localhost]
 
 PLAY RECAP *********************************************************************************
 localhost                  : ok=2    changed=1    unreachable=0    failed=0
 
 ```
 
-Check that the namespace was created:
-```bash
-$ kubectl get namespace
-NAME          	           STATUS    AGE
-default       	           Active    28d
-kube-public   		   Active    28d
-kube-system   	           Active    28d
-example-memcached          Active    3s
+Check that the ConfigMap was created:
+
+```none
+$ kubectl get configmaps
+NAME                    STATUS    AGE
+example-config          Active    3s
```

Comment:
all examples are based in the memcached as the sample; https://github.com/operator-framework/operator-sdk-samples/tree/master/ansible/memcached-operator

Can we please use it?

Reply:
Yes, that is an option. However, my feeling is that for this doc we don't want to force the users to clone a different repo (`git clone https://github.com/operator-framework/operator-sdk-samples.git`) to understand the tips.

The previous doc was using namespace creation as a simple example, which is good. I'm changing it to configmap because namespace is a cluster scoped resource which will fail the in-cluster testing, although everything works fine when running `operator-sdk run local`.

Reply:
My suggestion was NOT to use the sample. i do not think that we should link it.
just use the Memcached as an example. See the quick start, it will give the steps to create the Memcached sample. IMO we should stick with it instead of creating here `example-config `

Reply:
I will mark as solved because we have a comment on top about that already.